src/framework/platform/backends/sokol/graphics.zig

```diff
@@ -561,9 +573,10 @@ pub const ShaderImpl = struct {
         debug.info("Creating shader: {d}", .{graphics.next_shader_handle});
         const shader = sg.makeShader(shader_desc);
 
+        // TODO check this
```

Comment:
I believe this is correct looking at https://github.com/floooh/sokol/pull/1111/files#diff-5a4303cccd1516d570a9b7a605d2b5ce0487743f686ec7a5fef129b88efa8500